hack/externalnode/install-vm.ps1

```diff
@@ -174,34 +192,72 @@ function UpdateAgentConf() {
     }
 }
 
+function ClearOVSConfig() {
+    Log "Deleting OVS bridge $OVSBridge"
+    $AntreaSwitch = "antrea-switch"
+    Stop-Service $AntreaAgent
+    try {
+        $adapterName = (Get-VMNetworkAdapter -SwitchName $AntreaSwitch -ManagementOS).Name
+        ovs-vsctl.exe del-br $OVSBridge
+    }  catch {
+        Log "Failed to get VMSwitch $AntreaSwitch , rc $_"
+        exit 1
+    }
+
+    try {
+        Remove-VMSwitch -ComputerName $(hostname.exe) $AntreaSwitch  -Force
+    } catch {
+        Log "Ignore error while removing VMSwitch, rc $_"
+    }
+
+    try {
+        Rename-NetAdapter -Name "$adapterName~" -NewName "$adapterName"
+    }  catch {
+        Log "Failed to rename network adapter $adapterName~ to $adapterName, rc $_"
+        exit 1
+    }
+    Start-Service $AntreaAgent
+    Log "Done Deleting OVS bridge $OVSBridge"
+}
+
+function ConfigureAntreaMonitorService() {
+    Log "Copying install script to $AntreaAgentConfDir"
+    Copy-Item $myInvocation.PSCommandPath "$AntreaAgentConfDir\install-vm.ps1"
+    $Binary = (Get-Command Powershell).Source
+    $Arguments = "-ExecutionPolicy Bypass -NoProfile -File $AntreaAgentConfDir\install-vm.ps1 -ClearOVS true -OVSBridge $OVSBridge -Namespace $Namespace -BinaryPath $BinaryPath -ConfigPath $ConfigPath -KubeConfigPath $KubeConfigPath -AntreaKubeConfigPath $AntreaKubeConfigPath"
+    & nssm install $AntreaMonitor $Binary $Arguments
+    # Configure service to avoid restarts upon exit
+    nssm set $AntreaMonitor AppExit Default Exit
+}
+
 function ConfigureAntreaAgentService() {
     # Set environment variables
     [Environment]::SetEnvironmentVariable("NODE_NAME", $NodeName, [System.EnvironmentVariableTarget]::Machine)
     # Assume nssm is installed and configure service
     $AntreaAgentArgs = "--config $AntreaAgentConfPath --log_file $AntreaAgentLogFile --logtostderr=false"
-    log "Creating service $AntreaAgent $AntreaAgentPath $AntreaAgentArgs"
+    Log "Creating service $AntreaAgent $AntreaAgentPath $AntreaAgentArgs"
```

Comment:
Same as in previous comment. Service will be auto restart upon failure. This is the default configuration of nssm based service. Special handling needs to be done only for antrea-monitor service since we want to run it only once.